src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "razor-vscode-test",
+  "private": true,
+  "displayName": "Razor Unit Tests",
+  "engines": {
+    "vscode": "^1.31.0"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "yarn run clean && yarn run lint && tsc -p ./",
+    "lint": "tslint --project ./",
+    "postinstall": "node ./node_modules/vscode/bin/install",
+    "test": "jest",
+    "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand --colors"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.6",
```

Comment:
Can you make these have OSS approval?

Reply:
These are all versions from aspnet/Razor.VSCode and AspNetCore so they should all already be approved.